pkgs/development/libraries/plasma-wayland-protocols/default.nix

```diff
@@ -7,11 +7,11 @@
 
 mkDerivation rec {
   pname = "plasma-wayland-protocols";
-  version = "1.10.0";
+  version = "1.11.0";
 
   src = fetchurl {
     url = "mirror://kde/stable/${pname}/${pname}-${version}.tar.xz";
-    sha256 = "sha256-MZSIZ8mgRhPm3g0jrfy8Ws7N3vCzn5hrNF7GwZcnNv4=";
+    hash = "sha256-swPknCuAMOTrxs8cMy2G9yAVTDjBvKW/jojkx7MsbAs=";
   };
 
   nativeBuildInputs = [ extra-cmake-modules ];
```

Comment:
we could add https://invent.kde.org/libraries/plasma-wayland-protocols as a homepage

Reply:
I'll move it to the shared KDE helper infrastructure once that's merged.